frame/merkle-mountain-range/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+[package]
+name = "pallet-mmr"
```

Comment:
what about https://github.com/paritytech/substrate/blob/master/frame/merkle-mountain-range/Cargo.toml

Reply:
hmm... I tried to find this crate here https://github.com/paritytech/substrate/tree/polkadot-v0.9.34/frame
I'll try to use this version.